source/manual/datalabs-start-new-project.html.md

```diff
@@ -0,0 +1,164 @@
+- make sure secrets and credentials are usable locally, but kept out of version control
+- implement a consistent folder structure to reduce onboarding time
+
+If you use `govcookiecutter` to set up a data science project, `govcookiecutter` automatically covers most of these needs.
+
+See the [`govcookiecutter` documentation](https://github.com/ukgovdatascience/govcookiecutter) for more information.
+
+## Organise your workload using Trello
+
+GOV.UK Data Labs adopts [Agile ways of working](https://agilemanifesto.org/). We use Trello to maintain a team-wide [Kanban-style way of working](https://en.wikipedia.org/wiki/Kanban_(development)), although specific projects may adopt other methodologies such as [Scrum](https://scrumguides.org/index.html).
+
+To make sure all projects are visible to the team, you should write and update cards on Trello. GOV.UK Data Labs has two Trello boards:
+
+- the [Data Labs doing board](https://trello.com/b/FMptFIIw/data-labs-doing-board), a Trello board for all tasks in the current sprint
+- the [Data Labs planning board](https://trello.com/b/s6TGpjeA/data-labs-planning-board), a Trello board for all planned tasks for future sprints
+
+For new epics, create an epic using the __Epic template__ card and summarise the epic.
+
+For new and existing projects, create stories using the __Story template__ card.
+
+Stories should cover a specific task that has a valid acceptance criteria. Acceptance criteria are your “definition of done”. You should make the minimum developments to meet this criteria.
+
+Draft any further work over and above this acceptance criteria as new stories.
+
+### Label and estimate Trello cards
+
+For both epics and stories, make sure you label the Trello cards appropriately. If relevant, you should also try to label stories with a time estimate.
+
+Try to include time for colleagues to review your work. If stories are sufficiently small and discrete, reviewing should take a reasonable length of time.
+
+You should include time for colleagues to review because:
+
+- explicitly providing review time ensures reviewers have dedicated time for them
+- not planning sufficient review time may lead to poorer quality reviews due to non-story related pressures
+- any unused review time is never lost as reviewers can start other stories
+
+Try to make your stories manageable, and doable within a sprint (usually two weeks). This makes sure the story completion rate (otherwise known as velocity) is fast enough, and means that your stories are targeted to meet a need.
+
+If a story is not well defined, split up that story into smaller stories to make it easier to complete them.
+
+If it is difficult to estimate how long a story should take, consider running that story as a spike. A spike is a set time interval to explore and report back on a task.
+
+## Manage your machine's resources with AWS Sagemaker
```

Comment:
Arguably this could be in a different/new section, although it does feel OK in "start a new project"

Reply:
I'll put it into a separate page